ktor-samples/you-kube/src/you/kube/App.kt

```diff
@@ -41,7 +41,10 @@ fun Application.youKubeApplication() {
     }
 
     val key = hex("03e156f6058a13813816065")
-    val uploadDir = File("ktor-samples/you-kube/.video")
+    // val databaseConfig = environment.config.config("ktor.database")
+    // val location: String = File(databaseConfig.property("storage").getString()).resolve("h2").absolutePath
```

Comment:
Please remove irrelevant comment section, it can confuse people reading this code.